jobs/cloud_controller_ng/templates/stack_check.sh.erb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set +e
```

Comment:
**suggestion** Could we move `set +e` closer to where we don't want to exit, which is wrapped around the rake task?

Reply:
updating

Reply:
@Samze updated from offline conversation, should be the same result

```
#!/usr/bin/env bash

set -ex

CC_PACKAGE_DIR="/var/vcap/packages/cloud_controller_ng"
export BUNDLE_GEMFILE="${CC_PACKAGE_DIR}/cloud_controller_ng/Gemfile"
export STACKS_YML="/var/vcap/jobs/cloud_controller_ng/config/stacks.yml"
export NRCONFIG=/var/vcap/jobs/cloud_controller_ng/config/newrelic.yml

function stack_check {
  bundle exec rake stacks:stack_check
}

function main {
  stack_check
}

main

exit 0
```